website/views/shared/tasks/edit/dailies/repeat_options.jade

```diff
@@ -1,4 +1,6 @@
-legend.option-title=env.t('repeatEvery')
+legend.option-title
+  span.hint(ng-if='task.frequency=="weekly"', popover-title=env.t('repeatEveryHelpTitle'), popover-trigger='mouseenter', popover=env.t('repeatEveryHelpContent'))=env.t('repeatEvery')
+  span(ng-if='task.frequency!="weekly"')=env.t('repeatEvery')
```

Comment:
Wondering if there is a more elegant way to implement an if..else here. Suggestions?